browser/ui/browser_commands.cc

```diff
@@ -277,6 +277,11 @@ void ToggleSidebarPosition(Browser* browser) {
                     !prefs->GetBoolean(prefs::kSidePanelHorizontalAlignment));
 }
 
+void ToggleSidebar(Browser* browser) {
+  auto* brave_browser_window = BraveBrowserWindow::From(browser->window());
```

Comment:
please note how other hotkeys check `browser` value.
MacOS is picky about this, please make sure we don't crash like we did here: https://github.com/brave/brave-core/pull/16916

Reply:
Thanks for the advice! Updated - https://github.com/brave/brave-core/compare/afb1978a4f991ebadca657ea6267eacb6b0eed3f..36dd7422b865bfe3025844b98b3115e576ea5b46